c/include/KIM_ModelDriverCreate.h

```diff
@@ -175,12 +187,28 @@ void KIM_ModelDriverCreate_GetNumberOfParameterFiles(
  ** kim_model_driver_create_module::kim_get_parameter_file_name
  **
  ** \since 2.0
+ **
+ ** \deprecated As of 2.2.  Please use
+ ** KIM_ModelDriverCreate_GetParameterFileBasename() instead.
  **/
 int KIM_ModelDriverCreate_GetParameterFileName(
```

Comment:
@ellio167 I do not agree with deprecating this functionality. Many users and our model drivers do not care about the directory name and file base name, and it does not hurt to keep this functionality.